README.md

```diff
@@ -186,120 +202,15 @@ Your favorite IDE will display inline documentation for all OpenTK APIs. Additio
 
 Technical documentation about the implementation of OpenTK can be found in the [Technical Wiki](https://github.com/opentk/opentk/wiki).
 
-
 Need Help?
 ==========
 
-Post your questions at the [OpenTK forums](http://www.opentk.com/forum).
+Come chat with us on [Discord](https://discord.gg/GZTYR4s).
```

Comment:
It's already in first lines

Reply:
They're different; the top says "join discord to help us." the bottom says "join discord if you need help."